py/web/views/events.py

```diff
@@ -872,19 +872,25 @@ async def execute(self, m: Model):
 class EventUpdate(UpdateView):
     class Model(BaseModel):
         subject: constr(max_length=200)
-        message: str
+        message: Optional[str]
 
     async def check_permissions(self):
         await check_session(self.request, 'admin', 'host')
 
     async def execute(self, m: Model):
         event_id = await _check_event_permissions(self.request, check_upcoming=True)
+        payload_json = await self.request.json()
+        msgs = {k: v for k, v in payload_json.items() if (k.startswith('message'))}
+
+        if not len(msgs):
```

Comment:
```suggestion
        if not msgs:
```
this is python, not js :-)

Reply:
lol thanks